doc/backend/setting.md

```diff
@@ -302,15 +312,20 @@ Additional fields specific to tracim can be found in the `virtual_organization`
 - key_file: Path to the key file used for signing.
 - cert_file: Path to the certificate file used for signing.
 - xmlsec_binary: Path to the xmlsec1 binary for XML security operations.
-- metadata_cache_duration: Cache duration for remote metadata. In this example, the default cache duration is set to 86,400 seconds (1 day). 
+- metadata_cache_duration: Cache duration for remote metadata. In this example, the default cache duration is set to 86,400 seconds (1 day).
 - `virtual_organization`: Configuration for virtual organizations associated with IdPs. This is where you will define per-IdP settings. Each IdP is identified by its metadata URL.
   - `common_identifier`: A common identifier for the virtual organization associated with the IdP.
   - `logo_url`: URL to the organization's logo on the selection screen.
   - `displayed_name`: The displayed name of the organization on the selection screen.
   - `attribute_map`: Mapping of SAML attributes to specific names used within the SP.
+  - `profile_map`: Mapping of SAML attributes to tracim's user profile.
+    - `value`: Mapping of SAML attributes to the value that will be tested.
+    - `match`: Regex pattern that will be tested against value, if there is a match profile is set.
```

Comment:
waht syntax is used? is it real regex processed with python `re` module? If so, I suggest to explain it

Reply:
other question: when talking about this development, we talked about field combination. Is it working like this? If so, an example would be much appreciated

Reply:
It is regex processed with the python `re` module. I will make it clearer

Reply:
@lebouquetin I'm not sure to understand your second question

Reply:
is it possible to have a configuration which implements `public_name = "{firstName} {lastName}"`?

Reply:
Yes it is possible.